packages/manager/apps/telecom/src/app/telecom/pack/move/eligibility/lineNumber/move-eligibility-lineNumber.html

```diff
@@ -33,12 +33,154 @@
             </span>
         </span>
     </div>
-    <button
-        type="submit"
-        class="btn btn-primary"
-        data-translate="submit"
-        data-ng-disabled="$ctrl.loading || availabilityForm.phoneNumber.$invalid || $ctrl.method !== 'number'"
-    ></button>
-    <oui-spinner class="ml-2" data-ng-if="$ctrl.loading" data-size="s">
-    </oui-spinner>
+    <div class="mt-3">
+        <button
+            type="submit"
+            class="btn btn-primary"
+            data-translate="submit"
+            data-ng-disabled="$ctrl.loading || availabilityForm.phoneNumber.$invalid || $ctrl.method !== 'number'"
+        ></button>
+        <oui-spinner class="ml-2" data-ng-if="$ctrl.loading" data-size="s">
+        </oui-spinner>
+    </div>
+
+    <div data-ng-if="$ctrl.displayFirstStep" class="mt-3">
+        <div class="row">
+            <div class="col-md-6" data-ng-repeat="copper in $ctrl.copperOffers">
+                <div class="oui-box oui-box_light">
+                    <h4
+                        class="oui-box__heading"
+                        data-ng-if="copper.status === $ctrl.STATUS.active"
+                        data-translate="pack_move_eligibility_line_status_active"
+                    ></h4>
+                    <h4
+                        class="oui-box__heading"
+                        data-ng-if="copper.status === $ctrl.STATUS.inactive"
+                        data-translate="pack_move_eligibility_line_status_inactive"
+                    ></h4>
+
+                    <div data-ng-if="copper.errorMessage">
+                        <div
+                            data-translate="pack_move_eligibility_error_test"
+                        ></div>
+                        <div data-ng-bind="copper.errorMessage"></div>
+                    </div>
+                    <!-- Show address for eligibility and button to select this address -->
+                    <div data-ng-if="!copper.errorMessage">
+                        <div>
+                            <strong
+                                data-translate="pack_move_eligibility_address"
+                            ></strong>
+                            <div>
+                                <span
+                                    data-ng-bind="copper.endpoint.address.streetNumber"
+                                ></span>
+                                <span
+                                    data-ng-bind="copper.endpoint.address.streetName"
+                                ></span>
+                            </div>
+                            <div>
+                                <span
+                                    data-ng-bind="copper.endpoint.address.zipCode"
+                                ></span>
+                                <span
+                                    data-ng-bind="copper.endpoint.address.city"
+                                ></span>
+                            </div>
+                        </div>
```

Comment:
Maybe we can use the `address` tag in this case https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address